libs/net/dal/Config/ContentConfig.cs

```diff
@@ -0,0 +1,86 @@
+using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNetCore.Hosting;
+
+namespace TNO.DAL.Config;
+
+/// <summary>
+/// ContentConfig class, provides a way to configure storage of files.
+/// </summary>
+public class ContentConfig
```

Comment:
Don't duplicate code and functionality that doesn't need to be duplicated.  We already have a `StorageConfig` class that does all of this.

Reply:
I assumed that as you had asked me to reproduce the streaming endpoint in the ContentController that I should also reproduce supporting classes. I believe the streaming endpoint in the StorageController will stream from both storage and upload locations. I'm happy to do whatever fits with your original intentions.

Reply:
The StorageController endpoints were an example of how to get a stream to work.  Just remove duplicate where possible and reasonable.

Reply:
Removed this class.